docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -0,0 +1,267 @@
+    - The full digest for the image, with or without "sha256:".(**GWH_YS08 do we need the quotes and the colon on sha256?**)
+    - Environment: `Production` or `Integration`.
+    - Debug mode: `True` or `False`. (**GWH_YS09 Do they have to click any buttons or anything to generate the ID? (I can't get to this page)**)
+    
+      NOTE: Debug mode is only available in the integration environment. In Production, `False` is the only valid value.
+
+3. Save the `GCP Enclave ID` for use in the next step.
+
+2. Go to the [Enclave ID Management page](https://admin-integ.uidapi.com/adm/enclave-id.html) and provide the following values to register the GCP Enclave ID that you just generated:
+    - Name: enclave name
+    - Protocol: `gcp-oidc`
+    - Enclave ID: the `GCP Enclave ID` value that was generated in Step 2.
+
+(**GWH_YS10 I guess we don't need the above instructions but if so I need to clarify. For integration deployment does the partner ONLY do the below? Or do we have to do the above first, and if so, how does the partner know when we've done it? Not sure how it all fits together for the onboarding process.**)
+
+#### Create VM Instance 
+<!-- (for partner only) -->
+
+To create a VM instance, run the following, using your own valid values instead of the following placeholders:
+
+| Placeholder | Actual Value |
+| :--- | :--- |
+| `{INSTANCE_NAME}` | Your own valid VM name. |
+| `{SERVICE_ACCOUNT}` | The value you provided during account creation, in this format: `{SERVICE_ACCOUNT_NAME}@{PROJECT_ID}.iam.gserviceaccount.com`.<br/>For details, see [Confidential Space Account Setup](#confidential-space-account-setup) (Step 4). |
+| `{IMAGE_SHA}` | The SHA for your UID2 operator image digest, received during your UID2 account setup.<br/>For details, see [UID2 Operator Account Setup](#uid2-operator-account-setup). |
+| `{API_TOKEN}` | Your UID2 `api_token` value, received during your UID2 account setup, which identifies the private operator service.<br/>For details, see [UID2 Operator Account Setup](#uid2-operator-account-setup). |
+
+The following example uses placeholder values.
+
+```
+$ gcloud compute instances create {INSTANCE_NAME} \
+  --confidential-compute \
+  --shielded-secure-boot \
+  --maintenance-policy Terminate \
+  --scopes cloud-platform \
+  --image-project confidential-space-images \
+  --image-family confidential-space \
+  --service-account {SERVICE_ACCOUNT} \
+  --metadata ^~^tee-image-reference=ghcr.io/iabtechlab/uid2-operator@sha256:{IMAGE_SHA}~tee-restart-policy=Never~tee-env-DEPLOYMENT_ENVIRONMENT=integ~tee-env-API_TOKEN={API_TOKEN}
+```
+
+(**GWH_YS11 Is there any info we can provide re the results of running the above, how they can make sure it was successful?**)
+
+### Production Deployment
+
+To deploy a new UID2 Operator in the GCP Confidential Space Enclave into the integration environment, follow these steps.
+
+(**GWH_YS12 if we're not including the Integ steps I should put all the steps here.**)
```

Comment:
The diff betwwn Integ and Prod is quite small.
But we encourage partner to test/verify in Integ first then go to Prod.

Reply:
Noted, thx.